internal/web3ext/web3ext.go

```diff
@@ -766,7 +766,7 @@ web3._extend({
 	methods: [
 		new web3._extend.Method({
 			name: 'removeTransaction',
-			call: 'txpool_removeTransaction',
+			call: 'debug_removeTransaction',
```

Comment:
This method declaration needs to move to the `DebugJs` var.

Reply:
Looks like GitHub didn't index the changes properly. The declaration was transplanted to the `DebugJs` definition already: https://github.com/etclabscore/core-geth/pull/203/files#diff-743816a1d619b592f395ca0641e2f96df78708f35726f1099da4696a1a9384a0R476